livekit_models.proto

```diff
@@ -117,6 +117,8 @@ message VideoLayer {
   // target bitrate, server will measure actual
   uint32 bitrate = 4;
   uint32 ssrc = 5;
+  // publishing FPS for this layer, used with simulcasted streams
+  float fps = 6;
```

Comment:
thinking about using this with temporal layers?

Reply:
Why float?

Reply:
The intention is for ingress to reuse this structure when specifying desired layers. I think it'll be hard for clients to determine what fps temporal layers would have.

I had it as a float due to NTSC input streams having 29.97 fps. That's really the only reason :(. Any ideas on how we can better handle this case? GStreamer uses a fraction for fps. wdyt @cnderrauber @frostbyte73 ?

Reply:
Temporal layers are hard for server too :-) Have to add a frame rate measurement at temporal layer level to do it. I think fraction is fine. I just wanted to understand the reason behind a float. Not sure how exactly we can achieve fractional rate, but having it as a float is not an issue I think.

Reply:
the fraction is just as annoying imo, I had to do this to convert
https://github.com/livekit/server-sdk-go/blob/media/pkg/media/framerate/framerate.go